### DISCUSSION TITLE
Merge candles to one

### DISCUSSION THREAD

**Discussion by lyashkoandrei** [tradingview/lightweight-charts#1178] [Q&A]
Is it possible to automatically combine 15 1-minute candles into one 15-minute candle?

**Comment by adgower** [comment]
you need to take all 1 minute candles and check for highest and lowest

set time equal to first 1 minute candle time
set open equal to first 1 minute open
set high old high > new high ? old high else new high
set low old low < new low ? old low else new low
set close equal to last 1 minute close
